packages/frontend/src/pages/ParkingSpaces/components/create-applicant-for-listing/SyncInternalParkingSpaces.tsx

```diff
@@ -0,0 +1,175 @@
+          }}
+        >
+          Synka interna bilplatser
+        </Box>
+      </Button>
+      <Dialog onClose={onCloseModal} open={open} maxWidth="sm" fullWidth>
+        <Box paddingTop="1rem">
+          <Box display="flex">
+            <DialogTitle variant="h1" fontSize={24} textAlign="left">
+              Synka interna bilplatser från Xpand
+            </DialogTitle>
+            <Box
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              marginLeft="auto"
+              marginRight="1rem"
+            >
+              <IconButton onClick={() => setOpen(false)}>
+                <CloseIcon />
+              </IconButton>
+            </Box>
+          </Box>
+        </Box>
+        <DialogContent sx={{ paddingTop: '0.5rem' }}>
+          <Box minHeight="500px" display="flex" flexDirection="column">
+            <Box paddingX="0.5rem">
+              <Typography variant="body1">
+                Här kan du synkronisera alla <i>publicerade</i> bilplatser som
+                finns i Xpand till OneCore. När du trycker på "Synka" så hämtas
+                alla bilplatser från Xpand och sen läggs de in i OneCores
+                databas.
+              </Typography>
+              <Typography>
+                Om en bilplats som ligger inne i OneCore tas bort från Xpand så
+                kommer den behövas tas bort manuellt även här i
+                medarbetarportalen.
+              </Typography>
+            </Box>
+            <Box flex="1" marginTop="1rem" display="flex" overflow="scroll">
+              {syncInternalParkingSpaces.data && (
+                <SyncInternalParkingSpacesResult
+                  data={syncInternalParkingSpaces.data}
+                />
+              )}
+            </Box>
+            <LoadingButton
+              loading={syncInternalParkingSpaces.isPending}
+              variant="dark"
+              sx={{ marginTop: '1rem' }}
+              onClick={() => syncInternalParkingSpaces.mutate()}
+            >
+              Synka
+            </LoadingButton>
+          </Box>
+        </DialogContent>
+      </Dialog>
+    </>
+  )
+}
+
+const SyncInternalParkingSpacesResult = (props: {
+  data: InternalParkingSpaceSyncSuccessResponse
+}) => {
+  return (
+    <Box flex="1" display="flex" flexDirection="column">
+      <Typography fontWeight="bold" align="center">
+        Resultat synkronisering
+      </Typography>
+      <Box
+        maxHeight="500px"
+        overflow="scroll"
+        borderRadius="4px"
+        bgcolor="rgba(0,0,0,0.1)"
+        flex="1"
+        padding="1rem"
+      >
+        <Box>
+          <Typography component="span">Inlagda: </Typography>
+          <Typography component="span" fontWeight="bold">
+            {props.data.insertions.inserted.length}
+          </Typography>
+        </Box>
+        <Box>
+          <Typography component="span">Icke inlagda: </Typography>
```

Comment:
Like I wrote on slack, I think that failed inserts due to the data already being present in the db should not count as failed. But this is a question to bring in the team and not something we should handle in this PR :) Should be quite easy to just filter out listings with `active-listing-exists` right?